### ISSUE TITLE
isort skipping standard library file

### ISSUE THREAD

**Issue by Vinci08** [microsoft/vscode-isort#172]
Hi,

  I recently installed isort on VS Code on home desktop, but it didn't work when I tried to save file. Shft+Alt+O gave "no organize import action available." Here's the settings:
```
   "python.languageServer": "Jedi",
    "python.jediEnabled": true,
    "python.formatting.provider": "none",
    "security.workspace.trust.untrustedFiles": "open",
    "[python]": {
        "editor.wordBasedSuggestions": false,
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": true
        },
        "editor.defaultFormatter": "ms-python.black-formatter",
    },
    "isort.args":["--profile", "black"],
    "editor.codeActionsOnSave": {
        "source.organizeImports": true,
        "source.sortImports": true,
        "source.fixAll.markdownlint": true,
        "source.fixAll": true
    },
    "isort.check": true,
```

Here's the output from my log

```
c:\Users\xxx\Desktop\xxx\microservices\Scripts\python.exe -m isort --version-number
CWD Linter: c:\Users\xxx\Desktop\xxx\microservices
Version info for isort running for c:\Users\xxx\Desktop\xxx\microservices:
5.10.1

SUPPORTED isort>=5.10.1
FOUND isort==5.10.1

[Warn  - 6:17:41 PM] Skipping standard library file: c:\Users\xxx\Desktop\xxx\microservices\data\yyy.py
[Warn  - 6:17:59 PM] Skipping standard library file: c:\Users\xxx\Desktop\xxx\microservices\data\yyy.py
[Warn  - 6:21:55 PM] Skipping standard library file: c:\Users\xxx\Desktop\xxx\microservices\data\yyy.py

```

yyy.py is a python file I am currently working on and not really part of any standard library. Not sure why it's flagged as standard library file. Can you please provide some guidance on this, because I am so used with working with isort on my work laptop that this is really slowing me down.

**Comment by mikaeltulldahl** [comment]
I have the same problem

**Comment by brettcannon** [comment]
Is the file in your workspace or outside of it?

**Comment by Vinci08** [comment]
@brettcannon it is not within any workspace. The settings above is for user so that I can have isort available globally. 

In the same folder, I also tried to enable isort in Workspace settings, but again it just gave me same output about skipping standard file.

**Comment by brettcannon** [comment]
@Vinci08 I'm not concerned about the setting but where the file you have open and are trying to tweak the imports is. The warning seems to come from:

https://github.com/microsoft/vscode-isort/blob/ef37d377cab60abb0a25ce983b6b6591d186aec0/bundled/tool/server.py#L534

And that uses the following to figure out what's a stdlib module:

https://github.com/microsoft/vscode-isort/blob/ef37d377cab60abb0a25ce983b6b6591d186aec0/bundled/tool/utils.py#L50-L52

And the list of site paths comes from:

https://github.com/microsoft/vscode-isort/blob/ef37d377cab60abb0a25ce983b6b6591d186aec0/bundled/tool/utils.py#L30-L35

As you can see that list comes from Python itself.

And I just noticed something: did you put your source code _inside_ your virtual environment directory? It looks like `c:\Users\xxx\Desktop\xxx\microservices\` is a virtual environment and your code files are also in that directory. If that's true, then this makes sense as the virtual environment is considered a part of Python and thus anything in there is consider part of Python as well. If you move your code out of your virtual environment, then I bet that will solve the issue.

**Comment by Vinci08** [comment]
@brettcannon Thank you for the detailed explanation Brett. I indeed have my source code inside a venv directory. I checked with another file not part of any venv, and it works just like you suggested!

This led me to update isort interpreter path in User settings to a global python executable, which now works correctly in any virtual environment. This ticket can be closed now.

Thank you again for your time and guidance!
